app/i18n/en-US/sources.json

```diff
@@ -43,7 +43,7 @@
   "Set a goal for your viewers to help you reach.": "Set a goal for your viewers to help you reach.",
   "Follower Goal": "Follower Goal",
   "Twitch Follows": "Twitch Follows",
-  "Youtube Follows": "Youtube Follows",
+  "Youtube Follows": "Youtube Subscribers",
```

Comment:
This won't get translated into other languages unless you change the key (and all references to it).  I think this should be changed to be safe.